src/app/Home.jsx

```diff
@@ -0,0 +1,93 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { useState } from 'react';
+import axios from 'axios';
+
+// Allow a user to compare their NI contributions between 2018/19 and 2019/20
+export default function Home() {
+  const [income, setIncome] = useState(0);
+  const [ni2018, setNI2018] = useState(0);
+  const [ni2019, setNI2019] = useState(0);
+
+  const getNationalInsurance = async (runDate) => {
+    const { data } = await axios.post(
+      '/v1/national-insurance',
+      { income },
+      {
+        headers: {
+          'x-run-date': runDate,
+        },
+      }
+    );
+    return data.ni;
+  };
+
+  const compareNationalInsurance = async () => {
+    const niA = await getNationalInsurance('2018-04-06');
+    const niB = await getNationalInsurance('2019-04-06');
+
+    setNI2018(niA);
+    setNI2019(niB);
```

Comment:
There should be some consideration for error scenarios here - what happens if the call to the national insurance service fails? Consider wrapping this in a try/catch block and provide some feedback to the user letting them know something went wrong and what they can do (if anything) to resolve.